src/renderers/shared/fiber/isomorphic/ReactPortal.js

```diff
@@ -27,6 +27,7 @@ exports.createPortal = function(
   implementation: any,
   key: ?string = null,
 ): ReactPortal {
+  containerInfo._reactPortalContainer = {};
```

Comment:
> It seems like if we change this to be a boolean then it’s not a problem and we can always set it. Does it have to be an object?

Right - making it a boolean now, and so that no longer needs fixed. I guess I meant that in the case that it was an object we should do that.

I had thought that an object would be useful if we ever want to store more metadata on the portal container - but there is no reason to make it an object until then.

> I think we use unstable_ for actual APIs people might call. In this case something like __reactInternalIsPortalContainer would look better IMO.

That makes sense, thanks for the context! Will fix the name.

I'm still looking for where to assign the `__reactInternalIsPortalContainer` property, and will push another commit shortly that fixes that and the other issues. Thanks for the dialog about this.